public/locales/en/translation.json

```diff
@@ -41,6 +42,14 @@
     "siteMap": "Site Map",
     "copyright": "Copyright © 2020 State of California"
   },
+  "certification-header1": "What is certification?",
+  "certification-p1": "Certification is the required process of updating the EDD every two weeks with your unemployment status with basic eligibility information:",
+  "certification-list1-item1": "Are you physically able to work?",
+  "certification-list1-item2": "Are you available for work?",
+  "certification-list1-item3": "Are you ready and willing to accept work immediately?",
+  "certification-list1-item4": "Did you refuse work",
+  "certification-list1-item5": "Did you work and earn wages? If so, how much?",
+  "certification-p2": "The fastest way to certify is on <2>UI Online</2>. You can also do this by phone by calling 1-866-333-4606, or by mailing the paper form.",
```

Comment:
Minor: Should we make the phone number a tel: link?

Reply:
I think this is a great idea but need to merge this now, will create a ticket to add tel: links to all the phone numbers including this one!